src/docs/release/breaking-changes/text-field-material-localizations.md

```diff
@@ -0,0 +1,122 @@
+---
+title: TextField requires Material Localizations
+description: TextFields will now assert for Material Localizations.
+---
+
+## Summary
+
+Instances of `TextField` must have a `MaterialLocalizations` present.
+Trying to instantiate  a `TextField` without the proper localizations will
+result in an assertion such as the following:
+
+```
+No MaterialLocalizations found.
+TextField widgets require MaterialLocalizations to be provided by a Localizations widget ancestor.
+Localizations are used to generate many different messages, labels, and abbreviations which are used
+by the material library.
+To introduce a MaterialLocalizations, either use a MaterialApp at the root of your application to
+include them automatically, or add a Localization widget with a MaterialLocalizations delegate.
+The specific widget that could not find a MaterialLocalizations ancestor was:
+  TextField
+```
+
+## Context
+
+If your `TextField` is within the scope of a `MaterialApp`, the
+`DefaultMaterialLocalizations` is already be instantiated and may suit your needs without having
+to make a change to your existing code.
+
+If your `TextField` is not within a `MaterialApp`, you may provide the localizations of
```

Comment:
=> If the `TextField` doesn't descent from `MaterialApp`, you can use a `Localizations` widget to provide your own localizations.   (Don't use "may".  :) In this case, may is wrong because it's not really optional. )

Also, I would like to see example code here on how that would look.

Reply:
>Also, I would like to see example code here on how that would look.

There already is example code below illustrating this.